README.md

```diff
@@ -38,13 +38,19 @@ Simple yet powerful way to get your Rails API compliant with [JSON API](http://j
 
 Support:
 
-* Ruby ~> 2.0 with Rails ~> 4.0;
-* Ruby ~> 2.1 with Rails 5.
+* Ruby => 2.0 with Rails ~> 4.0;
```

Comment:
I think that sounds reasonable. It helps Future Developer know that they are not locked inside a particular MRI version.

It could look like:

```
* Ruby 2.x with Rails 4.x
* Ruby 2.2.2+ with Rails 5
```

This keeps it fuzzy for Rails 4.x and prevents any confusion about the [minimum required version for Rails 5](https://github.com/rails/rails/blob/v5.0.0/rails.gemspec#L10).

Would that work?